index.html

```diff
@@ -180,6 +180,7 @@ <h1>Welcome to the Enhanced Blackbox Explorer!</h1>
                         <a class="btn btn-primary btn-video-export" data-toggle="tooltip" title="Export your vide and chart setup to file"> Export video...</a>
                         <a class="btn btn-primary btn-workspaces-export" data-toggle="tooltip" title="Export your workspace configurations to file">
                             Export Workspaces...</a>
+                        <a class="btn btn-primary btn-csv-export" data-toggle="tooltip" title="Export your current graphs to CSV file"> Export CSV&hellip;</a>
```

Comment:
The other buttons use `...` and not `&hellip`. Please use `...` to let all the buttons with the same style. We can change all if it has some advantage but It does not use less space, so I can't see one.
![image](https://user-images.githubusercontent.com/2673520/42259888-9af6fc28-7f62-11e8-8d64-e2eb8f75d4b8.png)

Reply:
The only advantage is that using one single ellipse character instead of three dots is correct in this case. But, okay I can accept that people do such foolish things, eg. using `x` to indicate multiplication instead of `×` or the symbol of inch instead of quotation marks... :P